core/src/main/java/bio/terra/pearl/core/service/participant/ShortcodeUtilService.java

```diff
@@ -0,0 +1,47 @@
+package bio.terra.pearl.core.service.participant;
+
+import bio.terra.pearl.core.service.exception.internal.InternalServerException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+@Service
+public class ShortcodeUtilService {
```

Comment:
Generally, something is either a utility or a service.  I think of utilities as non-Spring components that encapsulate pure business logic.  So I'd rename this to ShortcodeService to connote that this hits the database  and has some spring stuff injected into it